### ISSUE TITLE
I really like the idea, and maybe able to contribute. However.....

### ISSUE THREAD

**Issue by adamchenwei** [xDae/styled-bootstrap#104]
Maybe its good to have all pending items list as issues, so people know how far this work goes and what's in progress, so to avoid dup works? Thanks!!!

**Comment by andyhennie** [comment]
sounds like a good idea.

**Comment by xDae** [comment]
Hi @adamchenwei!yes, my idea is to have all the components ready, but currently I have not so many free time, as an Open Source Project feel free to contribute

thanks!

**Comment by adamchenwei** [comment]
yea, absolutely! would you think its possible to break all the remaining work into issues so people can just take them on and keep them in track (like a JIRA ticket)? I think that will help people to contribute without worrying duplicate work others maybe already doing, since in considering there are already 9 forks and I see many changes happening! Let me know if it makes sense @xDae

**Comment by xDae** [comment]
yes, would be nice to have those remaining components as GH issues,

**Comment by andyhennie** [comment]
Do you have a list of the most popular forks? Would be nice to combine all features into one master repo.

**Comment by kopax** [comment]
Maybe you should have a look at #106, I've just posted an announcement for contributors.
